pkg/ibm/ibmp.go

```diff
@@ -114,26 +119,26 @@ func (r IBMPowerDynamicConfig) authenticatedService(ctx context.Context, kubeCli
 }
 
 func (r IBMPowerDynamicConfig) GetInstanceAddress(kubeClient client.Client, ctx context.Context, instanceId cloud.InstanceIdentifier) (string, error) {
-	log := logr.FromContextOrDiscard(ctx)
+	log := logr.FromContextOrDiscard(ctx).WithValues("platform", "ibm-p", "instanceId", instanceId)
 	service, err := r.authenticatedService(ctx, kubeClient)
 	if err != nil {
 		return "", err
 	}
 	ip, err := r.lookupIp(ctx, service, string(instanceId))
 	if err != nil {
-		log.Error(err, "Failed to lookup IP", "instanceId", instanceId, "error", err.Error())
+		log.Info("Failed to lookup IP", "error", err.Error())
 		return "", nil //todo: check for permanent errors
 	}
 	if err = checkAddressLive(ctx, ip); err != nil {
-		log.Error(err, "Failed to check address", "instanceId", instanceId, "error", err.Error())
+		log.Info("Failed to check address", "error", err.Error())
 		return "", nil
 	}
 	return ip, nil
 }
 
 func (r IBMPowerDynamicConfig) ListInstances(kubeClient client.Client, ctx context.Context, instanceTag string) ([]cloud.CloudVMInstance, error) {
-	log := logr.FromContextOrDiscard(ctx)
-	log.Info("Listing ppc instances", "tag", instanceTag)
+	log := logr.FromContextOrDiscard(ctx).WithValues("platform", "ibm-p")
```

Comment:
the `platform` keywork should probably be reserved to the `PLATFORM` parameter value and we should set it as soon as we know it and ensure its inherited by all logs from that point on.

The value here is probably better named "driver" or something else that is consistent with how we call these things in the code.

Reply:
yeah, I just forget what the code calls those.